_proposals/pup-0001.rst

```diff
@@ -0,0 +1,317 @@
+similar to how [Python Enhancement Proposals (PEPs)](https://www.python.org/dev/peps/pep-0001/)
+work, which are markdown documents that are also viewable/readable as rendered documents via a
+browser.
+
+
+Finding/Discovering Proposals
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+An index mimicking [the Python one](https://www.python.org/dev/peps/) would be created which would
+show all proposals in different states (active, approved, rejected, abandoned). These would also be
+viewable via pulpproject.org in a dedicated area.
+
+
+Proposal Process
+================
+
+The overall process is: Pre-proposal -> Authoring -> Announcement ->  Discussion and Feedback ->
+Revision -> Voting -> Decision
+
+Most of the process is facilitated using either the pulp-list or pulp-dev mailing lists. The
+following guidelines identify how they should.
+
+* For changes impacting users, [pulp-list](https://www.redhat.com/mailman/listinfo/pulp-list) is the
+  appropriate place.
+* For changes impacting developers, [pulp-dev](https://www.redhat.com/mailman/listinfo/pulp-dev) is
+  the appropriate place.
+
+Pre-Proposal
+^^^^^^^^^^^^
+
+Discussing an idea via the appropriate mailing list prior to writing a formal proposal is
+encouraged. This is intended to save the author time by gathering feedback quickly before writing a
+concrete, detailed proposal.
+
+
+Authoring
+^^^^^^^^^
+
+One or more authors create the proposal as a pull request against
+[the pulpproject.org repo](http://github.com/pulp/pulpproject.org/) using the next sequential
+proposal number. The proposal is a single file, e.g. `pup-0022.rst`. A core dev is expected to
+merge any proposal with adequate clarity and in the expected format.
+
+
+Announcement
+^^^^^^^^^^^^
+
+Once a proposal is merged to the proposals area, an announcement would be sent to the appropriate
+list, which would include a very brief introduction and a link to the proposal in its web viewable
+form.
+
+
+Discussion and Feedback
+^^^^^^^^^^^^^^^^^^^^^^^
+
+General discussion would occur on the announcement thread on the appropriate mailing list. Threaded
+e-mail readers should make this conversation format easy to follow, while keeping the proposal
+document in-tact and easy to read.
+
+
+Revision
+^^^^^^^^
+
+Revision would occur as additional pull requests against the active proposal. This allows the
+changelog of a proposal to always be maintained. This is how Python does it. This also allows
+edits/updates to come with a commit message indicating the concerns addressed or improvements
+including links to responses via the archive. Collaboration is also easy with a collaborator sending
+revisions to the author as a patch or a new pull request against the proposal.
+
+
+Voting
+^^^^^^
+
+The +1, +0, -0, -1 feedback process is used, with votes sent as responses via the
+announcement/discussion mailing list. The voting has the following meanings:
+
+* +1: "I love the idea and I’m strongly committed to it."
```

Comment:
I suggest changing these to focus on evaluating the proposal itself, and not on the voter's feelings. I think that mindset makes it easier for a participant to fully consider all options, with as much impartiality as is reasonable. Something like:

+1: Will benefit the project and should definitely be adopted.
+0: Might benefit the project and is acceptable.
-0: Might not be the right choice, but is acceptable.
-1: Not the right choice and should definitely not be adopted.

Suggestions on the above are of course welcome.

Put another way, this change supports the idea that we should be participating as custodians of the project, looking out for the project's best long-term interests. Sometimes that might even require choices we don't like, but can recognize as the best path.